apps/web/app/pages/portfolio/components/activity-list/index.tsx

```diff
@@ -16,19 +16,28 @@ export function ActivityList({ activity, isLoading }: ActivityListProps) {
   if (isLoading) {
     return (
       <Stack flexGrow={1} position="relative">
-        <Flex
-          p="space.06"
-          textAlign="center"
-          fontSize="24px"
-          height="360px"
-          justifyContent="center"
-        >
+        <Flex p="space.06" textAlign="center" fontSize="24px" justifyContent="center" flexGrow={1}>
           <LoadingSpinner />
         </Flex>
       </Stack>
     );
   }
 
+  if (activity.length === 0) {
+    return (
```

Comment:
We have a component for this in mobile that shows an image of a hammock - `ActivityEmpty`, it would be good to use a similar setup for that / ask for an empty design